docs/resources/resources-glossary.md

```diff
@@ -0,0 +1,132 @@
+
+See also: [application-specific API](#application-specific-api).
+
+## Contoso
+
+**Contoso** Ltd. (also known as Contoso and Contoso University) is a fictional company used by Microsoft as an example company and domain.
+
+## content add-in
+
+**Content add-ins** are surfaces that can be embedded directly into Excel or PowerPoint documents. Content add-ins give users access to interface controls that run code to modify documents or display data from a data source. Use content add-ins when you want to embed functionality directly into the document. See [Content Office Add-ins](../design/content-add-ins.md) to learn more.
+
+## custom function
+
+In the Office Add-ins documentation, a **custom function** is a user-defined function in Excel. Custom functions in Excel enable developers to add new functions, beyond the typical Excel features, by defining those functions in JavaScript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel. See [Create custom functions in Excel](../excel/custom-functions-overview.md) to learn more.
+
+## host
+
+In the Office Add-ins documentation, **host** typically refers to an Office application. The Office applications, or hosts, that support Office Add-ins are: Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [application](#application), [client](#client), [Office application, Office client](#office-application-office-client).
+
+## Office application, Office client
+
+In the Office Add-ins documentation, **Office client** refers to an Office application. The Office applications, or clients, that support Office Add-ins are: Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [application](#application), [client](#client), [host](#host).
+
+## platform
+
+In the Office Add-ins context, a **platform** usually refers to the operating system running an add-in. Platforms that support Office Add-ins are: Windows, Mac, iPad, and web browsers.
+
+## requirement set
+
+[!include[Requirement set note](../includes/office-js-requirement-sets.md)]
+
+## ribbon, ribbon button
+
+A **ribbon** is a command bar that organizes an application's features into a series of tabs or buttons at the top of a window. A **ribbon button** is one of the buttons within this series.
+
+## runtime
+
+In the Office Add-ins context, a **runtime** is a lifecycle, or the time during which an application is running.
+
+## task pane
+
+Task panes are interface surfaces that typically appear on the right side of the window within Word, PowerPoint, Excel, and Outlook. Task panes give users access to interface controls that run code to modify documents or emails, or display data from a data source. Use task panes when you don't need to embed functionality directly into the document. See [Task panes in Office Add-ins](../design/task-pane-add-ins.md) to learn more.
```

Comment:
I will leave this as an explicit list, because my understanding is that Excel, Outlook, PowerPoint, and Word are the only applications that support task panes (and that Project, OneNote, and Visio do not support task panes).